pykern/pkunit.py

```diff
@@ -129,6 +130,9 @@ def case_dirs(group_prefix=""):
     of these expect files is copmared to the corresponding `work_dir`
     actual file using `file_eq`.
 
+    Ignore_lines is a list of posix style regex. See `Posix style regex syntax
+    <https://www.gnu.org/software/findutils/manual/html_node/find_html/posix_002dbasic-regular-expression-syntax.html>_`
```

Comment:
Remove this, and specify Args: (below) and say:

ignore_prefix (iterable): `POSIX standard regular expressions <https://www.gnu.org/software/findutils/manual/html_node/find_html/posix_002dbasic-regular-expression-syntax.html>`_ to be passed to ``diff``

The syntax for rST is 

```rst
`phrase <link>`_
```